src/Card/CardMedia.js

```diff
@@ -34,13 +39,36 @@ export type Props = {
    * @ignore
    */
   style?: Object,
+  /**
+   * Component for rendering image.
+   */
+  component?: ElementType,
 };
 
 function CardMedia(props: DefaultProps & Props) {
-  const { classes, className, image, style, ...other } = props;
-  const composedStyle = { backgroundImage: `url(${image})`, ...style };
+  const { classes, className, image, style, component: ComponentProp, ...other } = props;
+  const composedStyle =
+    ComponentProp === 'div' ? { backgroundImage: `url(${image})`, ...style } : style;
+  const composedClassName = classNames(
+    {
+      [classes.root]: ComponentProp === 'div',
```

Comment:
What if component will be neither `div` nor `img`? I think in this case it should receive `classes.root` styles

Reply:
Thank you for review!

That's a proper point. I think `backgroundImage` should also be added to `style` prop in case anything else than `img` is specified.

Reply:
Thank you for contribution ;)